packages/python3.rb

```diff
@@ -147,5 +150,6 @@ def self.postinstall
     @piplist.drop(2).each do |pip_pkg|
       system "pip install #{pip_pkg} -U", exception: false
     end
+    puts "Please install tcl and tk with 'crew install tcl tk' if tkinter is needed.".lightblue unless ARCH == 'i686'
```

Comment:
Wouldn't it be easier just to make tcl and tk runtime instead of build dependencies?

Reply:
No because `tk` pulls in `libx11`, which in turn brings in LOTS of dependencies. (e.g. sommelier, etc) which just seems unnecessary, especially for a minimal buildessential `python3` case...

Reply:
Makes sense.